### DISCUSSION TITLE
Welcome to M07_UF4_practica20 Discussions!

### DISCUSSION THREAD

**Discussion by emirand20** [emirand20/M07_UF4_practica20#24] [Announcements]
## 👋 Welcome!
  We’re using Discussions as a place to connect with other members of our community. We hope that you:
  * Ask questions you’re wondering about.
  * Share ideas.
  * Engage with other community members.
  * Welcome others and are open-minded. Remember that this is a community we
  build together 💪.

  To get started, comment below with an introduction of yourself and tell us about what you do with this community.

**Comment by emirand20** [comment]
@rsobrinog <3.

**Reply by 4Tipy** [reply]
<3
